tests/test.js

```diff
@@ -108,3 +108,19 @@ describe('shim', function() {
 		delete Object.prototype[1];
 	});
 });
+
+describe('single function', function() {
+	var findAsFunction = function(func) {
+			return find.call(undefined, this, func, arguments[1]);
```

Comment:
Instead of creating `findAsFunction`, i think what we really want is just to use `find` directly here?